skymp5-server/cpp/server_guest_lib/MpActor.cpp

```diff
@@ -100,6 +213,13 @@ void MpActor::OnEquip(uint32_t baseId)
     VarValue args[] = { VarValue(std::make_shared<EspmGameObject>(lookupRes)),
                         VarValue::None() };
     SendPapyrusEvent("OnObjectEquipped", args, std::size(args));
+
+    if (baseId == 0x00064B43) {
+       AddItem(GetSlotItem(80, 10, 8, 2), 1);
```

Comment:
Instead of calling 4 times directly, we should call smth like `PieScript::Play(...)` to abstract these 4 lines.